manifests/config.pp

```diff
@@ -6,6 +6,7 @@
   $scrape_configs,
   $remote_read_configs,
   $remote_write_configs,
+  Boolean $use_camptocamp_systemd = $::prometheus::params::use_camptocamp_systemd,
```

Comment:
BTW, I don't think *subclasses* inheriting params.pp is a great pattern.  I would normally want to be able to override the parameter in the base class (in one place) and have that take effect in all the subclasses.  But this is consistent with the other parameters and fits the way the testing is currently setup.  (There's no `let(:pre_condition) { include prometheus }` stuff in there).  Also would anybody ever want/need to override the autodetection code in params.pp??